packages/origin-ui-core/src/components/Organization/OrganizationView.tsx

```diff
@@ -44,21 +48,45 @@ export function OrganizationView() {
     const classes = useStyles(useTheme());
 
     useEffect(() => {
-        const organization = userOffchain?.organization;
-        if (organization) {
-            const activeCountriesParsed: number[] = JSON.parse(organization.activeCountries);
-            setFormValues({
-                ...organization,
-                headquartersCountry: Countries.find(
-                    (c) => c.id === organization.headquartersCountry
-                ).name,
-                activeCountries: Countries.filter((c) => activeCountriesParsed.includes(c.id))
-                    .map((country) => country.name)
-                    .join(', '),
-                country: Countries.find((c) => c.id === organization.country).name
+
+        if (params.id == undefined) {
```

Comment:
Use == only if there is no other way. This check can be simplified if (!params.id)